Libraries/LibCrypto/SecureRandom.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2024, the Ladybird developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
```

Comment:
to make this header self-contained, you should include `<AK/Span.h>` here instead of in the cpp.